migration/migrator/migrations/course/20230828212638_update_past_materials_to_1900.py

```diff
@@ -0,0 +1,37 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("""
+        UPDATE course_materials
+        SET release_date =  make_timestamptz(1900, 1, 1, 0, 0, 0, 'America/New_York')
```

Comment:
what does to_timestamp do here?
without a comment this is non-intuitive

Reply:
We could do that, but I think we should be consistent with the other PR and choose 1900 for now.

Reply:
Since the 1900 time is already in the distant past, it doesn't matter too much if the timezone varies as the course material would still be released. That said, I do see how the `new Date(0)` is cleaner. Why is `new Date(0)` associated with 1970?

Reply:
1970 is fine for course materials and features in Submitty that use the date picker.
But if we in the future collect date-of-birth, this will be not sufficient.
Ideally we will use the same "beginning of time" for all date pickers.
(and stylize and implement features the same on all date pickers on the site).

If 1900 works with error, I suggest we do that now.

Reply:
Okay, if we're going to stick with 1900 then, then this PR is ready to be merged.